html/janus.js

```diff
@@ -1731,16 +1731,31 @@ function Janus(gatewayCallbacks) {
 			Janus.debug("  -- Audio tracks:", stream.getAudioTracks());
 			Janus.debug("  -- Video tracks:", stream.getVideoTracks());
 		}
+
 		// We're now capturing the new stream: check if we're updating or if it's a new thing
 		var addTracks = false;
-		if(!config.myStream || !media.update || (config.streamExternal && !media.replaceAudio && !media.replaceVideo)) {
-			config.myStream = stream;
+
+		if(!config.myStreams.length || !media.update || (config.streamExternal && !media.replaceAudio && !media.replaceVideo)) {
+            if(stream) {
```

Comment:
For sure here are some formatting issues. It seems that for intent you use spaces but the original code use tabs.

Reply:
Oh apologies, I will fix it as soon as I get time! 
Thanks for your feedback!